### DISCUSSION TITLE
Help when self hosting - docker swarm

### DISCUSSION THREAD

**Discussion by Paillat-dev** [supabase/supabase#27385] [Questions] [answered]
I have been trying to self host supabase with docker swarm for the past couple of days. I obviously encountered some issues due to compatibility, and the compose not originally being made to work with swarm. After fixing all theese, I have managed to get all the services running sucessfully, with all the healthchecks pasing. But when I open up the website, and load the default project, it loads infinitely:
![image](https://github.com/supabase/supabase/assets/75439456/5b9d7fcb-2bb1-4ffc-ac5d-779f667327a9)

The issue is probably not very big, but I am not familiar enough with supabase's structure to investigate more. Thus I would love if someone could help. You can find attached my docker compose, didn't want to paste it here since it is 400 lines long.
[docker-compose.yaml.txt](https://github.com/user-attachments/files/15901041/docker-compose.yaml.txt)

**Comment by mundia416** [comment]
facing the same issues

**Reply by Paillat-dev** [reply]
https://github.com/orgs/supabase/discussions/27467

**Comment by Paillat-dev** [comment, marked as answer]
https://github.com/orgs/supabase/discussions/27467
